tests/integration/test_storage_kafka/configs/kafka.xml

```diff
@@ -11,16 +10,39 @@
         <debug>cgrp,consumer,topic,protocol</debug>
 
         <!-- librdkafka stat in system.kafka_consumers -->
-				<!-- default 3000 (every three second) seems too long for test -->
-				<statistics_interval_ms>600</statistics_interval_ms>
+        <!-- default 3000 (every three second) seems too long for test -->
+        <statistics_interval_ms>600</statistics_interval_ms>
 
         <kafka_topic>
             <name>consumer_hang</name>
             <!-- default: 3000 -->
             <heartbeat_interval_ms>300</heartbeat_interval_ms>
+        </kafka_topic>
+        <consumer>
+          <auto_offset_reset>earliest</auto_offset_reset>
+            <kafka_topic>
+            <!-- Setting for topic will be applied only for consumer -->
+            <name>consumer_hang</name>
             <!-- default: 10000 -->
             <session_timeout_ms>6000</session_timeout_ms>
-        </kafka_topic>
+          </kafka_topic>
+          <kafka_topic>
+            <name>separate_settings</name>
+            <!-- default: 10000 -->
+            <session_timeout_ms>6001</session_timeout_ms>
+          </kafka_topic>
+        </consumer>
+        <producer>
+          <!-- NOTE: consumer_hang - name of the topic -->
+          <kafka_separate_settings>
```

Comment:
What is `kafka_separate_settings`? It does not look like a kafka topic (otherwise it would be wrapped in `<kafka_topic><name>`). Why do we need it? Why can't the producer settings simply be listed under `<producer>`?

Reply:
`<kafka_separate_settings>` this is the old syntax I left it here to check that the warning works fine.